### ISSUE TITLE
Raspberry pi cuberite multiplayer problems

### ISSUE THREAD

**Issue by luke1310** [cuberite/cuberite#5091]
Hi Guys,
I'm an absolute Newbie with the raspberry pi, and i tried to set up a cuberite Server. After some work i got it to run, but only for my network. I tried forewarding the port 25565, but my friend still can't join. I tried pretty much everything on the internet and it would be very nice if someone could help me. I could send some pictures or so, but at the moment i don't know, what would be important. Thanks for your help :)

**Comment by tigerw** [comment]
> After some work i got it to run

Was there anything particularly difficult or unintuitive that we could fix? What went wrong?

For port forwarding:

If you type `ip route show | grep default` in the terminal, you should get a line like
`none default via X.X.X.X dev wifi0 proto unspec metric 0`
that contains your router IP.

How to set up port forwarding depends on what router you have though, but you can post a screenshot of the config page. If you do also post the output of `ip addr show` on your Pi.

After you've set up the port forward, enter `curl icanhazip.com` in the terminal to get your public IP.

Try connecting to this public IP in your Minecraft client.

**Comment by luke1310** [comment]
It is possible to join the server, if you are in my Network. So local works. But if my friends want to play with me from home it doesn`t work.  I followed this tutorial for port forewarding: https://portforward.com/fritz-box/7490/ and did everything as it is in there, so i already know my router IP, and the IP of the Pi.

Update:

 I found my problem: I used the local IP, not the ip you get from curl icanhazip.com. Thank you for your help ;) Now it runs as i like it
